feat: fix mapbox controls

raphael-arce opened the pull request:
fix styling of map control buttons

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206746728905379

Jaszkowic reviewed (changes requested):
Looks nice! 

- Could you add max zoom level to the map? (`use-map-setup.tsx`, line 52 add `maxZoom: MAP_MAX_ZOOM_LEVEL`)

- Is it possible to add hover effects on desktop to the buttons?

Jaszkowic commented:
Also, is it possible to set the zoom level that is applied when on-geolocate is triggered? Right now it zooms out quite much. We could set it to `MAP_LOCATION_ZOOM_LEVEL` (defined in `use-map-constants.tsx`) as used for easing to a location search.

aeschi commented on src/index.css:
```diff
@@ -49,67 +49,45 @@
-.mapboxgl-ctrl button:not(:disabled):hover {
-	@apply bg-transparent;
+.mapboxgl-ctrl-group button:first-child {
+	@apply h-14 w-14 shadow;
```
I would keep using our defined shadows: shadow-gdk-hard

aeschi commented on src/index.css:
```diff
@@ -49,67 +49,45 @@
-	.mapboxgl-ctrl-icon {
-	@apply bg-[url('/images/icon-locate.svg')];
+.mapboxgl-ctrl-group button:last-child {
+	@apply h-14 w-14 shadow;
```
also here: shadow-gdk-hard

aeschi reviewed (changes requested):
already looks good!
If possible I would add a hover effect. Ideally the icon symbols should switch to gdk-light-gray on hover

raphael-arce commented:
@aeschi  I've added gdk-light-grey as hover bg color. I think it is a bit dark, but let me know what you think.

@Jaszkowic yes we agreed with Henriette to always put the controls on one side. I've added all your suggestions as well.

Jaszkowic reviewed (approved):
ship it 🚀
